core/chains/evm/client/rpc_client.go

```diff
@@ -601,16 +525,20 @@ func (r *rpcClient) HeaderByHash(ctx context.Context, hash common.Hash) (header
 	return
 }
 
-func (r *rpcClient) LatestFinalizedBlock(ctx context.Context) (*evmtypes.Head, error) {
-	return r.blockByNumber(ctx, rpc.FinalizedBlockNumber.String())
+func (r *RpcClient) LatestFinalizedBlock(ctx context.Context) (*evmtypes.Head, error) {
+	head, err := r.blockByNumber(ctx, rpc.FinalizedBlockNumber.String())
```

Comment:
why this change?

Reply:
I think this is a debugging artifact, will change back.